animations/lib/src/misc/repeating_animation.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class RepeatingAnimationDemo extends StatefulWidget {
+  static String routeName = '/misc/repeating_animation';
+
+  @override
+  RepeatingAnimationDemoState createState() => RepeatingAnimationDemoState();
+}
+
+class RepeatingAnimationDemoState extends State<RepeatingAnimationDemo>
+    with SingleTickerProviderStateMixin {
+  AnimationController _controller;
+  Animation<BorderRadius> _borderRadius;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _controller =
+        AnimationController(duration: const Duration(seconds: 2), vsync: this)
+          ..repeat(reverse: true);
```

Comment:
I may need to update some things of my own. I've been using status listeners for this.